### ISSUE TITLE
OpenXR integration

### ISSUE THREAD

**Issue by modulesio** [webmixedreality/exokit#880]
Exokit supports multiple headset backends, but it is ongoing effort to maintain them. This is a problem that [OpenXR](https://www.khronos.org/openxr) is intended to solve.

If Exokit implemented OpenXR, we could bring up more devices (notably, possibly Hololens) with less ongoing maintenance work.

**Comment by Palmer-JC** [comment]
With OpenXR 1.0 release, is this change the priority of this?
https://www.khronos.org/openxr/

**Comment by modulesio** [comment]
The main practical driving factor would be to bring up Windows Mixed Reality support.

@Palmer-JC do you know what the current support matrix of OpenXR is? I couldn't easily find it.

**Comment by Palmer-JC** [comment]
No, but [openXR](https://www.windowscentral.com/openxr-now-available-microsoft-store-windows-mixed-reality) is a free download from the Microsoft store, so I think there is no issue.

At the very least, would not go further down the path of stuff you could ditch with this.

**Comment by Palmer-JC** [comment]
Also, from the original link I posted, there is a [link ](https://docs.microsoft.com/en-us/windows/mixed-reality/openxr)which is more programmer oriented.  Even points to a Hololens emulator
